lib/src/client.dart

```diff
@@ -37,15 +39,25 @@ class Client {
     _cache = cache;
   }
 
+  set apiTokenPrefix(String value) {
+    _apiTokenPrefix = value;
+  }
+
   // Getters
   String get endPoint => this._endpoint;
 
   String get apiToken => this._apiToken;
 
-  Map<String, String> get headers => {
-        'Authorization': 'Bearer $apiToken',
-        'Content-Type': 'application/json',
-      };
+  String get apiTokenPrefix => this._apiTokenPrefix;
+
+  Map<String, String> get headers {
+    Map<String, String> h = new Map();
```

Comment:
Rename h to a descriptive name such as `headersMap `

Add content-type to the initialisation 
`Map<String, String> headersMap = { 'Content-Type':  'application/json'}`

New line in between variable declaration and if statement to separate code blocks